pyproject.toml

```diff
@@ -276,6 +276,9 @@ ignore = [
 [tool.ruff.flake8-pytest-style]
 fixture-parentheses = false
 
+[tool.ruff.pyupgrade]
+keep-runtime-typing = true
```

Comment:
This is actually incorrect. We are on Python 3.10 now, we don't have to keep runtime typing.

Reply:
Not setting this will cause mypy errors - might be a bug in Ruff's UPG rules.

Reply:
Aah ok :)